src/ontology/uberon-edit.obo

```diff
@@ -119536,7 +119536,7 @@ is_a: UBERON:0000105 ! life cycle stage
 relationship: part_of UBERON:0000068 ! embryo stage
 relationship: preceded_by RnorDv:0000010 {gci_relation="part_of", gci_filler="NCBITaxon:10116"}
 relationship: preceded_by UBERON:0000111 ! organogenesis stage
-relationship: preceded_by UBERON:0004707 ! pharyngula stage
+relationship: preceded_by UBERON:0004707 {gci_relation="BFO:0000066", gci_filler="NCBITaxon:7711"} ! pharyngula stage
```

Comment:
We have the shortcut for the material entity GCIs. Not sure if this will flip on a normalize? Just curious, not a blocker

Reply:
This will not flip to a shorthand form upon normalisation, because there is currently no `Typedef` frame for BFO:0000066 declared in `uberon-edit.obo`, so the `occurs_in` shorthand is not available to the OBO serialiser.